operations/categorical/go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/project-flogo/catalystml-flogo/operations/categorical
 
 require (
-	github.com/project-flogo/catalystml-flogo/action v0.0.0
-	github.com/project-flogo/core v0.9.0-rc.1
+	github.com/project-flogo/catalystml-flogo/action v0.0.0-20190919183915-24819a1a9a2b
```

Comment:
you could also use the tag for the repo... either way...

Reply:
We'll need to tag the action's mod actually, for that. So for now I just kept it as it is.